.gitignore

```diff
@@ -0,0 +1,4 @@
+/.idea/shelf/Shelve_idea_files_.xml
+/.idea/shelf/Shelve_idea_files_/shelved.patch
+/.idea/vcs.xml
+/.idea/workspace.xml
```

Comment:
This change was great, but consider adding the following line. 
`<View> Hello world </View>`